assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -619,7 +619,7 @@ che.wsagent.cors.enabled=true
 ## Factory defaults.
 # Editor and plugin which will be used for factories which are created from remote git repository
 # which doesn't contain any Che-specific workspace descriptors (like .devfile of .factory.json)
-che.factory.default_editor=org.eclipse.che.editor.theia:1.0.0
```

Comment:
Maybe it make sense to also update stacks?

Reply:
Good catch. Makes sense. Currently, some of the stacks use `1.0.0` and some `next`. Will update